CMakeLists.txt

```diff
@@ -118,7 +119,24 @@ foreach(SAMPLE ${SAMPLES})
     endif()
 
     add_dependencies(${SAMPLE_NAME} CopyZdf)
+
     if(WIN32)
         add_dependencies(${SAMPLE_NAME} CopyDlls)
     endif()
 endforeach()
+
+# TODO: Generlize how input file dependencies are copied, see issue #46
```

Comment:
If you don't mind, I suggest checking for the specific targets instead. That's what we actually want to check, the presence of Eigen and OpenCV is just coincidental. Something like this:
```cmake
# TODO: Generalize how input file dependencies are copied, see issue #46
add_custom_target(
    CopyHandEyeFiles
    COMMAND ${CMAKE_COMMAND} -E copy
            ${CMAKE_CURRENT_SOURCE_DIR}/HandEyeCalibration/robotTransform.yaml
            ${CMAKE_CURRENT_SOURCE_DIR}/HandEyeCalibration/handEyeTransform.yaml
            ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}
)
if(TARGET UtilizeEyeInHandCalibration)
    add_dependencies("UtilizeEyeInHandCalibration" CopyHandEyeFiles)
endif()
if(TARGET PoseConversions)
    add_dependencies("PoseConversions" CopyHandEyeFiles)
endif()
```